js/html/shared/HTML.svelte

```diff
@@ -314,14 +324,14 @@
 	});
 </script>
 
-{#if error_message}
+{#if html_error_message || css_error_message}
 	<div class="error-container">
 		<strong class="error-title"
 			>Error rendering <code class="error-component-name"
 				>{component_class_name}</code
 			>:</strong
 		>
-		<code class="error-message">{error_message}</code>
+		<code class="error-message">{html_error_message || css_error_message}</code>
```

Comment:
imo better to show both errors if there are both html and css errors, otherwise a user might think that the fix for the html error caused the css error.